src/scss/helpers.scss

```diff
@@ -28,7 +28,7 @@ small {
 }
 
 p {
-	margin-bottom: 12px;
+	margin-bottom: 0;
```

Comment:
Por que foi ajustado essa margin-bottom?

Reply:
Removi pq nas tags que utilizamos nao contamos com essa margem e sempre removemos. Como ja estamos atuando com essas mudanças gerais achei que poderia ser uma boa ja remover isso. 
É um problema? Se sim posso voltar e removo onde for usar.

Reply:
o arquivo em questão é importado no arquivo theme.scss que é importado globalmente no projeto, creio que seja de menos impacto mantermos os 12px mas com o token

Reply:
Isso, pode dar algum BO grande se removermos.

Reply:
Entendido. Vou voltar como estava e remover diretamente onde for usar. Mas acho que essa margem atrapalha o desenvolvimento. Talvez seja o caso de revermos esse uso